File: samples/c/sample1.c
#include <stdio.h>

int add_numbers(int a, int b) {
    return a + b;
}

int main() {
    int x = 5, y = 10;
    int sum = add_numbers(x, y);
    printf("Sum is: %d\n", sum);
    return 0;
}

File: samples/pascal/sample1.pas
program Sample1;
var
  x, y, sum: integer;

function AddNumbers(a, b: integer): integer;
begin
  AddNumbers := a + b;
end;

begin
  x := 5;
  y := 10;
  sum := AddNumbers(x, y);
  writeln('Sum is: ', sum);
end.
